app/lib/activitypub/activity/create.rb

```diff
@@ -147,6 +147,16 @@ def language_from_content
     @object['contentMap'].keys.first
   end
 
+  def object_url
+    return if @object['url'].blank?
+
+    value = first_of_value(@object['url'])
```

Comment:
Is first entry what we want to store? I don't know actual example, but we'd expect URL which can be resolved to activity json or HTML with link to those, I think.

Reply:
It's probably going to be fine. I don't see a use case for somebody to put a lot of different links into `url`. We could do a filter by `mimeType` but the array could be an array of strings or array of objects, and the objects could be missing `mimeType`, so I don't want to introduce the explosion of complexity here...